src/libs/antares/study/runtime/runtime.cpp

```diff
@@ -661,6 +663,19 @@ void StudyRuntimeInfos::initializeMaxClusters(const Study& study)
       = maxNumberOfClusters<CompareAreasByNumberOfClusters::renewable>(study);
 }
 
+void StudyRuntimeInfos::initializeInequalityBindingConstraints()
```

Comment:
Do we initialize, here ? I guess we rather store inequality binding constraints numbers, do we not ?
If so, I suggest we call this method **getInequalityBindingConstraintsNumbers()**. 
We do not need to store these numbers, so we just have to return them (vector<uint>)